music21/stream/makeNotation.py

```diff
@@ -1044,6 +1108,9 @@ def makeTies(
     if not returnObj:
         raise stream.StreamException('cannot process an empty stream')
 
+    if not common.isIterable(classFilterList):
+        classFilterList = [classFilterList]
```

Comment:
oh... I really think we need a `common.isIterable(..., *, streamsExcepted=False)` call -- I just keep seeing places (in this PR, in the Stream constructor, and all over the code) where we want to allow Streams...

OH WAIT!  It doesn't matter here, because we're not passing in a Stream, we'd be passing in the TYPE `stream.Stream` -- so no problem here.  Just thinking aloud.